bazel/external/quiche.BUILD

```diff
@@ -313,108 +415,103 @@ envoy_cc_test_library(
+    srcs = ["quiche/quic/core/quic_ack_listener_interface.cc"],
+    hdrs = ["quiche/quic/core/quic_ack_listener_interface.h"],
     copts = quiche_copt,
     repository = "@envoy",
     visibility = ["//visibility:public"],
     deps = [
+        ":quic_core_time_lib",
+        ":quic_core_types_lib",
         ":quic_platform_base",
-        ":quic_platform_export",
-        ":quic_platform_ip_address_family",
     ],
 )
 
 envoy_cc_library(
-    name = "quic_platform_socket_address",
-    srcs = ["quiche/quic/platform/api/quic_socket_address.cc"],
-    hdrs = ["quiche/quic/platform/api/quic_socket_address.h"],
-    copts = quiche_copt,
+    name = "quic_core_alarm_interface",
+    srcs = ["quiche/quic/core/quic_alarm.cc"],
+    hdrs = ["quiche/quic/core/quic_alarm.h"],
     repository = "@envoy",
     visibility = ["//visibility:public"],
     deps = [
-        ":quic_platform_export",
-        ":quic_platform_ip_address",
+        ":quic_core_arena_scoped_ptr_lib",
+        ":quic_core_time_lib",
     ],
 )
 
 envoy_cc_library(
-    name = "quic_platform_base",
-    hdrs = [
-        "quiche/quic/platform/api/quic_aligned.h",
-        "quiche/quic/platform/api/quic_arraysize.h",
-        "quiche/quic/platform/api/quic_bug_tracker.h",
-        "quiche/quic/platform/api/quic_client_stats.h",
-        "quiche/quic/platform/api/quic_containers.h",
-        "quiche/quic/platform/api/quic_endian.h",
-        "quiche/quic/platform/api/quic_error_code_wrappers.h",
-        "quiche/quic/platform/api/quic_estimate_memory_usage.h",
-        "quiche/quic/platform/api/quic_exported_stats.h",
-        "quiche/quic/platform/api/quic_fallthrough.h",
-        "quiche/quic/platform/api/quic_flag_utils.h",
-        "quiche/quic/platform/api/quic_flags.h",
-        "quiche/quic/platform/api/quic_iovec.h",
-        "quiche/quic/platform/api/quic_logging.h",
-        "quiche/quic/platform/api/quic_macros.h",
-        "quiche/quic/platform/api/quic_map_util.h",
-        "quiche/quic/platform/api/quic_mem_slice.h",
-        "quiche/quic/platform/api/quic_optional.h",
-        "quiche/quic/platform/api/quic_prefetch.h",
-        "quiche/quic/platform/api/quic_ptr_util.h",
-        "quiche/quic/platform/api/quic_reference_counted.h",
-        "quiche/quic/platform/api/quic_server_stats.h",
-        "quiche/quic/platform/api/quic_stack_trace.h",
-        "quiche/quic/platform/api/quic_str_cat.h",
-        "quiche/quic/platform/api/quic_stream_buffer_allocator.h",
-        "quiche/quic/platform/api/quic_string_piece.h",
-        "quiche/quic/platform/api/quic_string_utils.h",
-        "quiche/quic/platform/api/quic_uint128.h",
-        "quiche/quic/platform/api/quic_text_utils.h",
-        # TODO: uncomment the following files as implementations are added.
-        # "quiche/quic/platform/api/quic_fuzzed_data_provider.h",
-        # "quiche/quic/platform/api/quic_test_loopback.h",
-    ],
+    name = "quic_core_alarm_factory_interface",
```

Comment:
Should be quic_core_alarm_factory_interface_lib.

Can you go through this file and make sure all core library names are in sync? (looks like names end with "_interface" are different)